docker/Dockerfile.base.18.04

```diff
@@ -63,6 +74,9 @@ RUN mkdir -p /root/.mujoco && \
   ln -s $HOME/.mujoco/mujoco200_linux $HOME/.mujoco/mujoco200
   ENV LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/root/.mujoco/mujoco200/bin
 
+
+RUN python3.5 -m ensurepip && python3.5 -m pip install --upgrade pip
```

Comment:
how is this distinct from line 102 below?

Reply:
line 102 upgrades pip inside virtual env. This line installs basically replaces install of `python3-pip` package since it the python install doesn't come with pip. We use the system install of pip to do a system install of pipenv later

Reply:
can you leave a comment to this effect?